src/main/perl/CLI.pm

```diff
@@ -313,6 +313,9 @@ sub _initialize
     $self->init_history($self->option("history-instances"))
         if $self->option("history");
 
+    # Setup the verbose_logfile option
+    $self->setup_reporter(undef, undef, undef, undef, 1);
```

Comment:
i don't fully agree wrt the sizing, but i guess it can be done.
i propose to add the `verbose_logfile` option, default 0?

do you also want it configurable for ccm-fetch/ccm-initialise?